src/modules/auto-oembed.js

```diff
@@ -40,8 +40,9 @@
 	var embedOne = function (ctx, force) {
 		var vPlayer = ctx.find(PLAYER_SEL);
 		var autoLoad = vPlayer.attr('data-autoload');
-		
-		if (!force) {
+		var provider = vPlayer.attr('data-oembed-provider');
+
+		if (!force && provider.requiresVideo()) {
```

Comment:
provider its only a string here.. no requiresVideo function ;)

Reply:
I've rebased and fix it in c992f5c